pynbody/snapshot/ramses.py

```diff
@@ -794,7 +794,7 @@ def translate_info(sim):
 
     sim.properties['boxsize'] = sim._info['boxlen'] * l_unit
 
-    if sim._info['omega_k'] == sim._info['omega_l'] == sim._info['omega_b'] == 0.0:
```

Comment:
Hi,
The default value of `omega_b` in ramses is not zero anymore, so it is not relevant to include it in this test. It actually makes the units cosmological in non-cosmo simulations.